setup.py

```diff
@@ -155,6 +155,12 @@ def findsome(subdir, extensions):
     },
     cmdclass=cmdclass,
     package_data={
-        'reproman': []
+        'reproman':
+            findsome(opj("distributions", "tests", "files"), {"yml", "yaml"}) +
```

Comment:
Not for this PR, but it'd be nice if we'd decide a yaml prefix to use consistently across the codebase :]